assets/src/edit-story/components/form/radioGroup/index.js

```diff
@@ -71,8 +72,14 @@ const Helper = styled.div`
 
 function RadioGroup({ onChange, value: selectedValue, options }) {
   const radioGroupId = useMemo(() => uuidv4(), []);
+
+  // We need manual arrow key navigation here, as we have a global listener for those keys
+  // preventing default functionality.
```

Comment:
Maybe we should instead just remove `radio` from the `NON_EDITABLE_INPUT_TYPES`? This would allow the default behavior which would make sense in case of radios. WDYT?

Reply:
Hm, not a bad idea, will look into it.

Reply:
So this actually didn't work - and the same for the buttons. Not completely sure why, maybe something about the events not bubbling up correctly.

Reply:
Hmm, odd, when I test locally (on the main branch), it seems to work when just removing `radio` from `NON_EDITABLE_INPUT_TYPES`. E.g. I remove this, click on "Status and Visibility" Panel, tab into the radio button, use the arrow keys and the values change.

Is it not working at all when you try or it working in an unexpected way?